.travis.yml

```diff
@@ -4,20 +4,34 @@ python:
   - "2.7" #what we built it for originally
   - "nightly" #whatever 3.x is hemorrhaging-edge
 
+services:
+  - postgresql
+
+before_script:
+  - pip install psycopg2 pymongo requests sqlalchemy
+  - export GREASE_DB_USER="postgres"
+  - export GREASE_DB_PASSWORD=""
+  - export GREASE_PKG_LOADER="tgt_grease_"
+  - export GREASE_VERBOSE_LOGGING=1
```

Comment:
Nice minimal install. Eventually we will want to leverage the grease.conf file for this